rewrite-javascript/rewrite/test/javascript/format/format.test.ts

```diff
@@ -185,4 +186,22 @@ describe('AutoformatVisitor', () => {
                  `import {delta, gamma} from 'delta.js'`)
             // @formatter:on
         )});
+
+    test('anonymous function expression', () => {
+        return spec.rewriteRun(
+            // @formatter:off
+            //language=typescript
+            typescript(
+                `const fn = function () {return 99;};`,
+                 // TODO the space after `const fn=` is excessive
+                 `
+                const fn = 
```

Comment:
If we can get rid of that trailing space on this line, that would also be nice. I think we have a setting that requires spaces around `=`, but I guess a `\n` should also be accepted.